examples/kubeflow_pipelines_orchestration/README.md

```diff
@@ -104,33 +96,86 @@ pipeline on a local Kubeflow Pipelines deployment.
+
+The stack recipe is called `k3d-modular` and is available in the ZenML
+[stack recipe repository](https://github.com/zenml-io/mlops-stacks/tree/main/k3d-modular).
+The recipe is modular, meaning that it can be configured to use different
+orchestrators, Model Deployers, and other tools. By default, it uses the
+following tools:
+
+* The **Minio Artifact Store** as a remote alternative to the local file system
+for storing artifacts, the Minio server is deployed as a Kubernetes pod.
+* The docker images that are created to run your pipeline are stored in a local
+docker **Container Registry**.
+* The **Kubeflow Orchestrator** is responsible for running your ZenML pipeline
+in Kubeflow Pipelines.
+* The **MLFlow Expirement Tracker** is used to track the experiments and
+metrics of your pipeline runs. The MLFlow server is deployed as a Kubernetes
+pod.
+
+```bash
+zenml stack recipe deploy k3d-modular
+```
+>**Note**:
+> This recipe comes with MLflow, Kubeflow and Minio enabled by default. If you want any other components like KServe, Seldon or Tekton, you can specify that using the `--install/-i` flag.
 
-## 📄 Infrastructure Requirements (Pre-requisites)
+#### 🏁 Deploy the stack using ZenML CLI
 
-You don't need to set up any infrastructure to run your pipelines with Kubeflow, locally. However, you need the following tools installed:
-  * Docker must be installed on your local machine.
-  * Install k3d by running `curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash`.
+The flow to get started for this example can be the following:
 
-## Create a local Kubeflow Stack
+1. Pull the `k3d-modular` recipe to your local system. Learn more about what 
+this recipe does from its README.
 
-To get a stack with Kubeflow and potential other components, you can make use of ZenML's Stack Recipes that are a set of terraform based modules that take care of setting up a cluster with Kubeflow among other things.
+```shell
+zenml stack recipe pull k3d-modular
+```
 
-Run the following command to deploy the local Kubeflow Pipelines stack:
+2. (Optional) 🎨 Customize your deployment by editing the default values in the `locals.tf` file.
 
-```bash
+3. Deploy the stack using the ZenML CLI:
+
+```shell
 zenml stack recipe deploy k3d-modular
 ```
->**Note**:
-> This recipe comes with MLflow, Kubeflow and Minio enabled by default. If you want any other components like KServe, Seldon or Tekton, you can specify that using the `--install/-i` flag.
 
-This will deploy a local Kubernetes cluster with Kubeflow installed. 
-It also generate a stack YAML file that you can import as a ZenML stack by running 
+> **Note**
+    > This command can also automatically import the resources created as a ZenML stack for you. Just run it with the `--import` flag and optionally provide a `--stack-name` and you're set! Keep in mind, in that case, you'll need all integrations for this example installed before you run this command.
```

Comment:
Shouldn't this be the default way of doing things? Seems much easier than the separate stack import + stack set below

Reply:
That actually can make it way easier from a UX perspective. probably would ask @wjayesh about this and if we can make it part of this PR?